packages/manager/modules/support/src/tickets/new-ticket/new-ticket.html

```diff
@@ -69,3 +69,6 @@
                 data-on-click="$ctrl.goToTickets()">
     </oui-button>
 </oui-message>
+
+<support-tickets-new-feedback
```

Comment:
why creating a component here? it looks super overkill, it's just a link

Reply:
I was expecting to have the link at more than one place but everything is in one page 😅 

Then I thought it could be migrated to a full component if it's ever exported to its own

Reply:
The component is now used at multiple places as it was really ugly to have the links below loading screen